doc-content/jbpm-docs/src/main/asciidoc/BPMN2/design-business-process-proc.adoc

```diff
@@ -116,4 +116,12 @@ public class MyListener implements ProcessEventListener {
 --
 
 . In the process designer canvas, use the left toolbar to drag and drop BPMN components to define your business process logic, connections, events, tasks, or other elements.
++
+[NOTE]
+====
+A task and event in {PRODUCT} expects one incoming and one outgoing flow. If you want to design a business process with multiple incoming and multiple outgoing flows, use gateways (recommended) or set the JVM system property `jbpm.enable.multi.con` to `true`.
+
+When `jbpm.enable.multi.con` property is enabled for Java virtual machine (JVM), then ensure that the business process that JVM executes contains the similar system property as enabled otherwise, the process engine throws an exception.
```

Comment:
Not sure I understand where the business process should contain the similar system property. Isn't it enough to set it for JVM?

Reply:
@MarianMacik As per the discussion on: https://issues.redhat.com/browse/RHPAM-2656

jbpm.enable.multi.con is not just a property on JVM, but the process must also have the similar property. 

Please correct if I am wrong.

Reply:
@hmanwani-rh I don't see any references to process property in that discussion. The only reference is that it is a server-side (JVM) property.